### DISCUSSION TITLE
Profiler of React Developer Tools not supported

### DISCUSSION THREAD

**Discussion by medihack** [blitz-js/blitz#2852] [Q&A]
When using Blitz (in dev mode), the Profiler of the React Developer Tools (Chrome extension) tells me that `Profiling is not supported` because `Profiling support requires either a development or production-profiling build of React v16.5+.`. Is this because Blitz uses an alpha version of React? Is there a workaround?

**Comment by flybayer** [comment]
wow that's a bummer, because the alpha react version definitely supports it. Their detection logic is faulty. If you aren't using suspense, you can revert to React v17. 

Otherwise open an issue on the react dev tools repo. Probably it's even an easy PR to fix this.
